### ISSUE TITLE
Times logged should be UTC

### ISSUE THREAD

**Issue by benlangfeld** [cschneid/irclogger#3]
It would be useful to log messages in UTC and provide timezone offset in the views.

I would do this myself, but having looked through the code, I can't find the actual logger bot. The code in the bot branch appears to be non-production.

**Comment by cschneid** [comment]
Correct, the bot code in the branch is a mostly-done, but not live reworking of the bot code I have.  The live site is still running the perl bot I borrowed from the #perl6 channel.  (ilbot).  

I'd like to finish up the ruby version of the bot, but it keeps getting put off.

**Comment by cschneid** [comment]
Also, the specific issue is valid. I'd like to do that, and have a settings page on irclogger.com

**Comment by cschneid** [comment]
Followup as a note to myself, tasks here are:

1) Settings page
2) Cookie based settings handler (jquery magic!)
3) Display times in correct offset (all JS I suppose... maybe using query string to be easier & do it server-side).
